### DISCUSSION TITLE
What Type of Applications?

### DISCUSSION THREAD

**Discussion by rcmsoftwaredevelopment** [serenity-is/Serenity#5403] [Help]
Hi folk, I know a lot of work is going on underneath the surface of serenity but I will be building an application soon and I don't know if to use this framework or asp.net zero.  I was just wondering what commercial applications people are using this for and if possible any links?  My issue with serenity is due to the lack of progress on the user interface since I used it last back in 2017.  

Thanks Ray

**Comment by minhhungit** [comment]
I think you should use this framework instead asp.net zero, Serenity theme is not really "beautiful", modern like asp.net zero so your users might not like it, but it is CLEAN, SCIENTISLY and VERY GOOD UX

If I have to choice, I pick Serenity

I worked with Serenity from 2017 and applied it for many projects

**Reply by edwardch** [reply]
In terms of being able to deliver value quickly and efficiently to your users (who are your clients) I would pick Serenity hands down.

It makes development work easy so you can focus more time talking to users/clients about the stuff that really matters to them, allowing you to grow. 

In terms of look and feel, I personally feel that if your app brings value to your users they will care less about UI. Some people say the UI needs a bit of work but I think it's very easy for your users to understand use your app.

That said, Serenity & Serene have had a lot of thought and hard work put into it during the .NET 5 upgrades, including a more modular approach to the front end. You can already style it however you please, and going forward it will only get easier.

I have been a happy user (and very happy pro subscriber) for a few years now.


While I'm here - big thanks to the core Serenity team for all the hard and dedicated work over the past few months. There is a lot that we don't get to see behind all those commits - you are amazing! I'm here to stay.

**Reply by erehtolleh** [reply]
I agree with @edwardch I have been using Serenity since asp.net 2.0 something.  Things I like about it:
1. I like the file structure, easy to find and understand, easy to learn how everything is related.
2. Very easy to use for applications where you need a lot of digital forms.  At least that is what I use it for.  Integrate approval into the forms.
3. I have an applications that has been running very smoothly since day one.  Never had an issue with Serenity itself.  Users never complaints about it.  The only time it goes down is when network goes down.
4. Easy to generate the forms.  You just have to focus on users requirements and modify to meets the user's requirements.  You don't have to worry about the CRUD.
5. I am sure there are more but trying to go to sleep

I am not sure if this is even comparable to asp.net zero that you are referring to.  Serenity is built from asp mvc and now .net core.  You can do whatever asp/.net/razor stuff you want within Serenity.  They are the same framework just configured a little differently and then a lot more with Serenity core, Sergen...
